hoot-core/src/main/cpp/hoot/core/io/OsmApiDbReader.cpp

```diff
@@ -138,7 +140,9 @@ NodePtr OsmApiDbReader::_resultToNode(const QSqlQuery& resultIterator, OsmMap& m
       _defaultCircularError,
       resultIterator.value(ApiDb::NODES_CHANGESET).toLongLong(),
       resultIterator.value(ApiDb::NODES_VERSION).toLongLong(),
-      resultIterator.value(ApiDb::NODES_TIMESTAMP).toUInt()));
+      OsmUtils::fromTimeString(resultIterator.value(ApiDb::NODES_TIMESTAMP)
```

Comment:
OK. So I copied this code from HootApiDBReader. And it works, but I don't like it. There is no reason to convert to a string, then back from a string. I'm going to change it...